### DISCUSSION TITLE
What scenarios require the use of `Combobox.List`/`Select.List`?

### DISCUSSION THREAD

**Discussion by Jonas-C** [chakra-ui/zag#1666] [Q&A]
They are not documented on the zag site, but they're used in a couple of examples. Should you always use them, or just in a few scenarios? If the latter, when should they be used?

**Comment by anubra266** [comment]
@Jonas-C I'd say the importance depends on your concern for accessibility. See the attributes for Select List for example here:

https://github.com/chakra-ui/zag/blob/main/packages/machines/select/src/select.connect.ts#L452
